docs/package.json

```diff
@@ -18,6 +18,7 @@
     "@docusaurus/preset-classic": "2.4.1",
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.2.1",
+    "docusaurus": "^1.14.7",
```

Comment:
You don't need to add this dependency for docsaurus to work, could you please remove it

Reply:
Done